PubnativeLite/PubnativeLite/Interstitial/HyBidInterstitialAd.h

```diff
@@ -31,6 +31,7 @@
 - (void)interstitialDidTrackImpression;
 - (void)interstitialDidTrackClick;
 - (void)interstitialDidDismiss;
+- (void)interstitialDidFinish;
```

Comment:
Why do we need in the interstitial ad? Makes sense for rewarded but not for generic interstitial, since it could just be an html creative.

Reply:
@eros902002 I've added it just to have one additional control point. Maybe we will need sometime detecting if the ad playing finished or not. But if you think we will not need it, then I can delete that part. But for me it was like a good functionality to have :)

Reply:
It's a good idea to have such a method. Maybe we can have it as a separate delegate? What do you think @can-soykarafakili ? The problem is that if we add it to the current Interstitial delegate then it becomes not backwards compatible. All old integration would have to update adapters and SDK implementation before updating to a the HyBid version.

Reply:
@eros902002 if there will be any problems, then I can delete that part

Reply:
I think old integrations will have a warning saying that all HyBidInterstitialAdDelegates are not implemented. I guess it wont raise any errors but definitely will have warnings @eros902002 @orkhanalizada2020

Reply:
If having warnings wont hurt anyone, I think we are okay with this additional method 👍